conf/defaults.ini

```diff
@@ -1191,6 +1191,32 @@ ha_redis_peer_name =
 # The maximum number of simultaneous redis connections.
 ha_redis_max_conns = 5
 
+# Enable TLS on the client used to communicate with the redis server. This should be set to true
+# if using any of the other ha_redis_tls_* fields.
+ha_redis_tls_enabled = false
+
+# Path to the PEM-encoded TLS client certificate file used to authenticate with the redis server.
+ha_redis_tls_cert_path =
```

Comment:
Are these required, or can you leave them blank and the client will use the certificates installed in the OS?

Whichever the answer is, might be worth noting in the doc comment